src/main/java/cn/nukkit/block/BlockLight.java

```diff
@@ -0,0 +1,68 @@
+package cn.nukkit.block;
+
+import cn.nukkit.item.Item;
+import cn.nukkit.math.AxisAlignedBB;
+import cn.nukkit.utils.Identifier;
+
+public class BlockLight extends BlockTransparent {
+
+    public BlockLight(Identifier identifier) {
+        super(identifier);
+    }
+
+    @Override
+    public void setDamage(int meta) {
+        super.setDamage(meta & 0xF);
+    }
+
+    @Override
+    public int getLightLevel() {
+        return getDamage() & 0xF;
+    }
+
+    @Override
+    public AxisAlignedBB getBoundingBox() {
+        return null;
```

Comment:
This is likely going to cause NPEs somewhere else.

Reply:
Returning null in getBoundingBox means that the block does not show any line when we target it. 
[BlockAir](https://github.com/NukkitX/Nukkit/blob/2.0/src/main/java/cn/nukkit/block/BlockAir.java#L51-L53) and [BlockLiquid](https://github.com/NukkitX/Nukkit/blob/2.0/src/main/java/cn/nukkit/block/BlockLiquid.java#L80-L83) does the same.

[BlockTripWireHook](https://github.com/NukkitX/Nukkit/blob/2.0/src/main/java/cn/nukkit/block/BlockTripWire.java#L42-L44) and [BlockTripWire](https://github.com/NukkitX/Nukkit/blob/2.0/src/main/java/cn/nukkit/block/BlockTripWire.java#L42-L44) also returns null there but I suspect they are wrong because the block have collision box in game.

I inspected the code and every call to `getBoundingBox()` has a null check before using it. 
I think that if a NPE is thrown somewhere because it returned null then the call site should be fixed to handle null returns as `getBoundingBox()` is nullable.

Reply:
Rectifying BlockTripWireHook and BlockTripWire might be right returning null because even thought they have a visual bounding box, it doesn't stop the player movement.  [FloodableBlock](https://github.com/NukkitX/Nukkit/blob/2.0/src/main/java/cn/nukkit/block/FloodableBlock.java#L41-L45) also does that but making `recalculateBoundingBox()` returns null, which would result in the same effect as the default implementation of `getBoundingBox()` calls `recalculateBoundingBox()`